src/css/onboarding.css

```diff
@@ -0,0 +1,30 @@
+#container {
+    display: flex ;
+    justify-content: center ;
+    height: 100% ;
+    background-color: grey ;
+}
+
+#panelImage {
+    box-shadow: 5px 5px 15px 15px dimgrey ;
+    max-width: 90vw ;
+    height: 80vh ;
```

Comment:
I think this should be changed to `max-height` to keep things from stretching as well.